tests/kafkatest/services/performance/consumer_performance.py

```diff
@@ -77,9 +77,6 @@ def __init__(self, context, num_nodes, kafka, topic, messages, version=DEV_BRANC
         assert version.consumer_supports_bootstrap_server() or (not new_consumer), \
             "new_consumer is only supported if version >= 0.9.0.0, version %s" % str(version)
 
-        assert version < V_2_0_0 or new_consumer, \
```

Comment:
`new_consumer` is always `true`, right?

Reply:
Hi @chia7712, thanks for the suggestion. I'm curious whether we can assume `new_consumer` is always `true` in `ConsoleConsumer`? Thanks.

Reply:
yes and we can remove the `new_consumer` as follow-up

Reply:
https://issues.apache.org/jira/browse/KAFKA-17977